components/Tabs/src/MDCTabBar.h

```diff
@@ -90,8 +90,16 @@ IB_DESIGNABLE
  */
 @property(nonatomic, nonnull) UIColor *unselectedItemTintColor UI_APPEARANCE_SELECTOR;
 
-/** Ink color for taps on tab bar items. Default: Semi-transparent white. */
-@property(nonatomic, nonnull) UIColor *inkColor UI_APPEARANCE_SELECTOR;
+/**
+ Determines if an @c MDCTabBar should use the @c MDCRippleView behavior or not.
+ By setting this property to @c YES, @c MDCRippleView is used to provide the user visual
+ touch feedback, instead of the legacy @c MDCInkView.
+ @note Defaults to @c NO.
```

Comment:
Same here similarly to https://github.com/material-components/material-components-ios/pull/7596#discussion_r293630410